examples/project_llm_fine_tune/project_llm_fine_fune/assets.py

```diff
@@ -0,0 +1,395 @@
+import dagster as dg
+import pandas as pd
+from collections import defaultdict
+import time
+from dagster_openai import OpenAIResource
+import dagster_duckdb as dg_duckdb
+from . import utils
+import json
+from textwrap import dedent
+from collections import Counter
+from openai import OpenAI
+from typing import Iterable
+import project_llm_fine_fune.constants as constants
+
+
+goodreads = dg.AssetSpec(
+    "Goodreads",
+    description="Goodreads Data. https://mengtingwan.github.io/data/goodreads#datasets",
+    group_name="raw_data",
+)
+
+
+@dg.asset(
+    kinds={"duckdb"},
+    description="Goodreads graphic novel data",
+    group_name="processing",
```

Comment:
Since we're getting it from the http endpoint, I might consider this `ingestion`